build/entire/entire.pkg

```diff
@@ -33,9 +33,9 @@ driver/audio					0.5.11	require		S
 driver/crypto/dca				0.5.11	require		S
 driver/crypto/tpm				0.5.11	require		S
 driver/firewire					0.5.11	require		S
-driver/graphics/agpgart				0.5.11	require		S
+driver/graphics/agpgart				0.5.11	optional	S
```

Comment:
The idea was that make it possible to use the illumos `driver/graphics/agpgart` and `driver/graphics/drm` or the one from the gfx-drm repository. So the user is able to remove the package and install a different one if needed.

Reply:
Ah ok - I had thought that the new ones would just replace those from `illumos-gate` but if there's a reason that someone might want those then we need to have a solution. It's not reasonable to have two `driver/graphics/drm` packages with the same version and just different timestamps, the user would not know the difference (which is presumably why you increased the version number in the original request).

```
bloody%  pkg contents -m incorporation/jeos/illumos-gate | grep drm
depend fmri=driver/graphics/drm@0.5.11,5.11-151027.0 type=incorporate
```

We have two incorporations that cover packages from illumos-omnios (I don't know why) but `jeos/illumos-gate` constrains the version of these packages to 0.5.11. You've taken care of that through `illumos-gate.exclude`.

The other `consolidation/osnet/osnet-incorporation` does not actually include these two packages - I wonder if that's because OpenIndiana wanted to be able to replace them.

It was done here - https://github.com/illumos/illumos-gate/commit/9c31cf024c08128fce3fce725716f43a0286b787

>Make it easier to provide the agp and drm packages from a different gate by setting "noincorp" on these packages.

So, I retract my previous comment - please use a version number higher than 0.5.11 for these but you do not need to change the line in entire to `optional`, require is fine as it only specifies a minimum version. By default the gfx-drm versions will be used but a user can downgrade and `pkg freeze` if they want the gate ones.